libs/community/langchain_community/retrievers/bedrock.py

```diff
@@ -115,13 +115,16 @@ def _get_relevant_documents(
         results = response["retrievalResults"]
         documents = []
         for result in results:
+            content = result["content"]["text"]
+            result.pop("content")
+            if "score" not in result:
+                result["score"] = 0
+            if "metadata" in result:
+                result["sourceMetadata"] = result.pop("metadata")
```

Comment:
Can you use snake case instead of camel case here.

```suggestion
                result["source_metadata"] = result.pop("metadata")
```

Reply:
With the approach that we have taken to populate the metadata here, if we add a new field in the [retrieve API](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_Retrieve.html#API_agent-runtime_Retrieve_ResponseElements) in the future that will get added to the metadata automatically and that field name will be in camelCase. So having sourceMetadata in snake case will bring an inconsistency in the namings. We would like this integration to work out of the box for new additions in the bedrock APIs. Hence we should keep camelCase in my opinion.